### PR TITLE
Fix render Document Temples from Documents

### LINKED ISSUE #7173: Opening portal templates from documents broken
### Describe the bug
I broke this when I refactored portal. Will also restyle documents UI while I here.

### Client configuration

- Browser:
- OpenEMR version: master 7.0.2
- Operating system:

### PR DESCRIPTION
**Pull request by sjpadgett**
Since portal refactor the render portal documents from patient documents has been broken. Nuts on me!
Figured I'd do a bit of restyle while here.

Fixes #7173 

#### Changes proposed in this pull request:
refactor controller to call new portal document menu fix calling requested document render from patient documents restyle documents to be cleaner

### CODE REVIEW

**Comment by sjpadgett** [comment]
image

File: portal/patient/templates/OnsiteDocumentListView.tpl.php [hunk 1]
```diff
@@ -82,7 +83,7 @@
         ";var autoRender=" . js_escape($auto_render) . ";var auditRender=" . js_escape($audit_render) . ";var renderDocumentName=" . js_escape($auto_render_name) .
         ";var catid=" . js_escape($category) . ";var catname=" . js_escape($catname) . ";</script>";
     echo "<script>var recid=" . js_escape($recid) . ";var docid=" . js_escape($docid) . ";var isNewDoc=" . js_escape($isnew) . ";var newFilename=" . js_escape($new_filename) . ";var help_id=" . js_escape($help_id) . ";</script>";
-    echo "<script>var isPortal=" . js_escape($is_portal) . ";var isModule=" . js_escape($is_module) . ";var webRoot=" . js_escape($webroot) . ";var webroot_url = webRoot;</script>";
+    echo "<script>var isPortal=" . js_escape($is_portal) . ";var isModule=" . js_escape($is_module) . ";var webRoot=" . js_escape($webroot) . ";var doc_edit = doc_edit;var webroot_url = webRoot;</script>";
```
Comment (bradymiller) [inline portal/patient/templates/OnsiteDocumentListView.tpl.php:86]: isn't doc_edit a php variable (if so, then need to do the `. js_escape...` to it like the other php variables on this line)

**Comment by bradymiller** [comment]
dance-bear

File: portal/patient/templates/OnsiteDocumentListView.tpl.php (hunk 1 above)
Comment (sjpadgett) [reply portal/patient/templates/OnsiteDocumentListView.tpl.php]: fixed thanks for catch

Comment (sjpadgett) [reply portal/patient/templates/OnsiteDocumentListView.tpl.php]: I was going to deprecate this feature but after receiving emails from folks that I broke it on portal refactor I took another look and better integrated with patients available docs.
Also gave me a chance to fix the document smarty and better organise the code so there is that.
I guess I'm pleased and probable should document this stuff better so more folks knows how useful this feature can be.

Will merge so I can clean up and backport my current fixes and build a hot patch. If anyone needs something cherry picked let me know and I'll do.

**Pull request merged**